### PR TITLE
Content wip sentence casing disability eligibility

### PR DESCRIPTION
**Pull request by DanielleSoCompany**
## Page to edit
url: All pages within /disability/eligibility/

## Origin of request (internal/stakeholder/user feedback)
Style guide decision to move to sentence casing

## Description of what's needed (edits/link changes/additions)
See instructions in ticket: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/16779

### CODE REVIEW

**Comment by DanielleSoCompany** [comment]
**Note:** Please do not merge this PR. Content team will review and merge. Thanks!

**Comment by ncksllvn** [comment]
Hey @DanielleSoCompany, I just resolved a conflict with the disability/index.md that occurred after deleting the duplicate links yesterday. Please take a look to make sure I did it right. Thanks!

**Comment by RLHecht** [comment]
@bethpotts heading info has been added to all the disability files

File: pages/disability/eligibility/hazardous-materials-exposure/agent-orange/c-123-aircraft.md [hunk 1]
```diff
@@ -75,19 +76,89 @@ You can upload supporting documents along with your online application, includin
...
+  <li>USAF Form 5 (aircraft flight duties)</li>
+  <li>USAF Form 781 (aircraft maintenance duties)</li>
+  <li>Dependency records (marriage certificate and children’s birth certificates)</li>
+  <li>Medical evidence (like a doctor’s report or medical test results)</li>
+</ul>
+
+</div>
+
+</li>
+</ol>
```
Comment (ncksllvn) [inline pages/disability/eligibility/hazardous-materials-exposure/agent-orange/c-123-aircraft.md]: This page fails our accessibility tests -

```
Failed [fail]: (http://localhost:3001/disability/eligibility/hazardous-materials-exposure/agent-orange/c-123-aircraft/: [serious] <li> elements must be contained in a <ul> or <ol>
```

@bethpotts @DanielleSoCompany Can you take a look at this? I think the floating `</ol>` on line 101 and the `<li class="process-step list-four" markdown="0">` can be removed, but I'm not sure.

Comment (bethpotts) [reply pages/disability/eligibility/hazardous-materials-exposure/agent-orange/c-123-aircraft.md]: @ncksllvn I think I fixed it. Looked like some random content got thrown in somehow. I removed it. Let me know if all looks good now.

Comment (bethpotts) [reply pages/disability/eligibility/hazardous-materials-exposure/agent-orange/c-123-aircraft.md]: I think the errant content may have been meant for the online application, not this static page.

**Pull request merged**